gnmi/README.md

```diff
@@ -1,22 +1,37 @@
-# Fake gNMI Service (central datastore)
+# gNMI Service Reference Implementation (central datastore)
 
-`gnmit.GNMIServer` serves as a central datastore to which subscribing
-"tasks/agents" can be attached.
+`gnmi.GNMIServer` serves as the reference implementation of gNMI for lemming and the central datastore to which reconcilers can be attached.
 
-There are two ways to subscribe to the central datastore:
+## External Clients
 
-## 1 Subscribing using a `gnmit.Task` interface
+External clients use gNMI to configure and read state of the reference device.
+## Internal Reconcilers
```

Comment:
Can you provide a brief English definition of reconcilers to introduce the term? So something like "a reconciler is a set of initialization methods and long-running methods (either of which are optional) that monitor the intended configuration, or other internal state, and reconciles the current operational state of the device to reach the intended state".